components/restapi/core/accounts.go

```diff
@@ -62,10 +61,10 @@ func congestionByAccountAddress(c echo.Context) (*api.CongestionResponse, error)
 func validators(c echo.Context) (*api.ValidatorsResponse, error) {
 	var err error
 	pageSize := restapi.ParamsRestAPI.MaxPageSize
```

Comment:
Maybe we should add and use the function from the indexer.

```go

func pageSizeFromContext(c echo.Context) uint32 {
	maxPageSize := uint32(deps.RestAPILimitsMaxResults)
	if len(c.QueryParam(api.ParameterPageSize)) > 0 {
		pageSizeQueryParam, err := httpserver.ParseUint32QueryParam(c, api.ParameterPageSize, maxPageSize)
		if err != nil {
			return maxPageSize
		}

		if pageSizeQueryParam < maxPageSize {
			// use the smaller page size given by the request
			maxPageSize = pageSizeQueryParam
		}
	}

	return maxPageSize
}
```

Reply:
Or we should even add this function to the httpserver package, so we can reuse it in the indexer as well.

Reply:
I opened PR in inx-app: https://github.com/iotaledger/inx-app/pull/115